kickstarter/android-oss
# NTV-364: [Regression bug] Creator dashboard not loading for release 3.7.0

# 📲 What

- Creator dashboard was not loading

# 🤔 Why

There was a change on the rewards model field `estimatedDeliveryOn` compatible with GraphQL but not with V1, the creator dashboard still uses V1 API, and `gson` library to parse the json response to our data model, this force us to not be able to change the data time of any field required on V1 calls or the parsing process will fail.

# 🛠 How
- You can see in the detailed message the issue
![Screenshot 2022-12-06 at 2 42 02 PM](https://user-images.githubusercontent.com/4083656/206047707-ccb0f23a-57c3-4640-b884-b964b67f403e.png)

- the UI was always in loading state and not crashing due to the `apiErrorOperator` avoiding it to display back any error message or crashing.
![Screenshot 2022-12-06 at 2 45 07 PM](https://user-images.githubusercontent.com/4083656/206047878-950f0a5a-a029-4c66-b571-ee6ee5a340d4.png)

# 👀 See

| Before 🐛 |

https://user-images.githubusercontent.com/4083656/206048029-203dae7d-6a4b-46a1-b974-bb3292992f5c.mp4



| After 🦋 |

https://user-images.githubusercontent.com/4083656/206048057-17b8cd62-fb43-44fe-885c-35d938102145.mp4




| --- | --- |

|  |  |

# 📋 QA

- Load the creator Dashboard
- Go to the project https://www.kickstarter.com/projects/alstup/the-first-connected-round-counter-and-timer-in-the-world all the estimate delivery dates for rewards should be "December"

# Story 📖

[NTV-634](https://kickstarter.atlassian.net/browse/NTV-634)
